cmd/objectstore/objectstore_create.go

```diff
@@ -39,6 +39,22 @@ var objectStoreCreateCmd = &cobra.Command{
 
 		if bucketSize == 0 {
 			bucketSize = 500
+		} else if bucketSize < 500 {
```

Comment:
I would condensate the two conditions (bucketSize == 0) and (bucketSize < 500)  in the same condition (bucketSize < 500)